extensions/wikia/ContributionAppreciation/ContributionAppreciationCommonHooks.class.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+class ContributionAppreciationCommonHooks {
+	const TRACKING_URL = 'https://beacon.wikia-services.com/__track/special/appreciation_email';
+	const EMAIL_CATEGORY = 'ContributionAppreciationMessage';
+
+	public static function onSendGridPostbackLogEvents( $events ) {
+		foreach ( $events as $event ) {
+			if ( self::isAppreciationEmailEvent( $event ) ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)